TelemetryTests/TelemetryTests.swift

```diff
@@ -1,36 +0,0 @@
-//
-//  TelemetryTests.swift
-//  TelemetryTests
-//
-//  Created by Steph Leroux on 2017-04-07.
-//  Copyright © 2017 Mozilla. All rights reserved.
```

Comment:
Ah, good. Looks like this commit cleans up a bunch of stuff in the previous; do fold before landing.